applications/convection_diffusion/rotating_hill/application.h

```diff
@@ -180,6 +206,11 @@ class Application : public ApplicationBase<dim, Number>
 
         // hypercube volume is [left,right]^dim
         dealii::GridGenerator::hyper_cube(tria, left, right);
+
+        unsigned int const frequency   = 1;
+        double const       deformation = (right - left) * 0.1;
+        apply_deformed_cube_manifold(tria, left, right, deformation, frequency);
+
```

Comment:
Note tha we use `this->param.mapping_degree = this->param.degree` now.
Hence, we test the case, where the mapping is higher-order, but stationary.
AMR does not yet support `MappingDoFVector` (`MappingQCache`), which is the next step.
For this, we need a strategy to get the DoF vector from the `MappingDoFVector`, attach it to the `SolutionTransfer`, interpolate and update the `MappingQCache` after executing AMR.
This is for a future PR.